cpp/CMakePresets.json

```diff
@@ -6,6 +6,30 @@
     "patch": 0
   },
   "configurePresets": [
+    {
+      "name": "_allocator-none",
```

Comment:
Can we use this in `features-minimal` too?

Reply:
Hmm, yes we can. I was trying to add some specialty to `_` presets, that is, not using it in regular inheritance where there is no conflicting variables. However I think nothing functional really prevents it from being used in regular inheritance. I'll try that and update soon.

Reply:
Addressed.